tools/tertiary-analysis/data-scxa/atlas_import_experiment_data.xml

```diff
@@ -0,0 +1,141 @@
+<tool id="atlas_import_experiment_data" name="Atlas import: get experiment data" version="@TOOL_VERSION@+galaxy0"  profile="@PROFILE@">
```

Comment:
if this is replacing retrieve-scxa, I would suggest that it uses the same file name, tool id and possibly tool name than the previous version, with a good bump in version.